change/@microsoft-fast-foundation-841d2e2b-ff3a-42d1-bd3c-e143b9f12afd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "doc fix",
```

Comment:
This will show in the changelog even though it's a none type - can we add a little more detail here so it's clear what this pertains to in the history?